includes/sanitizers/class-amp-blacklist-sanitizer.php

```diff
@@ -93,24 +110,68 @@ private function sanitize_a_attribute( $node, $attribute ) {
 			// rev removed from HTML5 spec, which was used by Jetpack Markdown.
 			$node->removeAttribute( $attribute_name );
 		} elseif ( 'target' === $attribute_name ) {
-		       if ( '_blank' === $attribute->value || '_new' === $attribute->value ) {
-			       // _new is not allowed; swap with _blank
-			       $node->setAttribute( $attribute_name, '_blank' );
-		       } else {
-			       // only _blank is allowed
-			       $node->removeAttribute( $attribute_name );
-		       }
+			// _blank is the only allowed value and it must be lowercase.
+			// replace _new with _blank and others should simply be removed.
+			$old_value = strtolower( $attribute->value );
+			if ( '_blank' === $old_value || '_new' === $old_value ) {
+				// _new is not allowed; swap with _blank
+				$node->setAttribute( $attribute_name, '_blank' );
+			} else {
+				// only _blank is allowed
+				$node->removeAttribute( $attribute_name );
+			}
+		} elseif ( 'href' === $attribute_name ) {
+			// If the href starts with a '/', append the home_url to it for validation purposes.
```

Comment:
It's not a bug, Google does not want you to use `//` URLs. They want explicit protocols and I don't consider that a bug.

However, this isn't that. Relative URLs are totally allowed. To validate them, I'm just appending the home url so they can be tested. There's a corresponding unit test for this that's passing called 'a_with_href_relative'. I think we're OK here.